mlflow/models/cli.py

```diff
@@ -200,32 +193,11 @@ def build_docker(model_uri, name, env_manager, mlflow_home, install_mlflow, enab
     'python_function' flavor.
     """
     env_manager = env_manager or _EnvManager.VIRTUALENV
-    _get_flavor_backend(model_uri, docker_build=True, env_manager=env_manager).build_image(
+    build_docker_api(
         model_uri,
         name,
+        env_manager=env_manager,
         mlflow_home=mlflow_home,
         install_mlflow=install_mlflow,
         enable_mlserver=enable_mlserver,
     )
-
-
-def _get_flavor_backend(model_uri, **kwargs):
```

Comment:
Also needs to be called by `mlflow.models.build_docker`, so move into `mlflow.models.flavor_backend_registry` and import it from `mlflow.models.cli` and `mlflow.models`.